src/components/Theme/tokens.scss

```diff
@@ -21,46 +40,77 @@
-  --color-secondary-button-text: var(--color-primary);
-  --color-secondary-button-border: var(--color-primary);
-  --color-secondary-button-border-hover: var(--color-primary-hover);
-  --color-secondary-button-border-active: var(--color-primary-active);
-
-  --color-icon-button: $color-transparent;
-  --color-icon-button-hover: rgba(var(--ods-color-neutral-400));
-  --color-icon-button-active: rgba(var(--ods-color-neutral-600));
-
-  /* - Camera button colors */
-  --color-camera-button: rgba(var(--ods-color-neutral-white));
-  --color-camera-button-hover: var(--color-primary-button-hover);
-  --color-camera-button-active: var(--color-primary-button-active);
-
-  /* - Selectors button colors */
-  --selectors-box-shadow-hover: 0 0 0 2px var(--color-primary-hover);
-  --selectors-box-shadow-active: 0 0 0 2px var(--color-primary-active);
-
-  /* Other button variables */
-  --button-border-radius: var(--ods-border-radius-medium);
-  --button-group-vertical: false;
+  /* Primary Button customisations */
+  --osdk-color-content-button-primary-text: rgba(
+    var(--ods-color-content-on-action)
+  );
+
+  --osdk-color-background-button-primary: rgba(
+    var(--ods-color-background-action)
+  );
+  --osdk-color-background-button-primary-hover: rgba(
+    var(--ods-color-background-action-hover)
+  );
+  --osdk-color-background-button-primary-active: rgba(
+    var(--ods-color-background-action-active)
+  );
+  --color-background-button-primary-disabled: rgba(
+    var(--ods-color-neutral-300)
+  );
+
+  --osdk-color-border-button-primary: rgba(var(--ods-color-border-action));
+
+  /* Secondary Button customisations */
+  --osdk-color-content-button-secondary-text: rgba(
+    var(--ods-color-content-action)
+  );
+
+  --osdk-color-background-button-secondary: rgba(
+    var(--ods-color-background-action-subtle)
+  );
+  --osdk-color-background-button-secondary-hover: rgba(
+    var(--ods-color-background-action-subtle-hover)
+  );
+  --osdk-color-background-button-secondary-active: rgba(
+    var(--ods-color-background-action-subtle)
+  );
+
+  --osdk-color-border-button-secondary: rgba(var(--ods-color-border-action));
+
+  /* Button group customisations */
+  --osdk-border-radius-button: var(--ods-border-radius-medium);
+  --osdk-button-group-stacked: false;
+
+  /* Document Type Selectors option button colors */
+  --color-background-doc-type-button: rgba(var(--ods-color-neutral-white));
+
+  --color-border-doc-type-button: rgba(var(--ods-color-border-input));
+  --color-border-doc-type-button-hover: rgba(
+    var(--ods-color-border-action-hover)
+  );
+  --color-border-doc-type-button-active: rgba(
+    var(--ods-color-border-action-focus)
+  );
+
+  /* Back, Close (modal) icon button colors */
+  --color-background-button-icon-hover: rgba(var(--ods-color-neutral-400));
+  --color-background-button-icon-active: rgba(var(--ods-color-neutral-600));
+
+  /* Selfie Camera button colors */
```

Comment:
These are not documented elsewhere, is it expected?

Reply:
They don't have `--osdk` prefix so I think they're internal